src/libs/actions/IOU.js

```diff
@@ -482,7 +482,10 @@ function getMoneyRequestInformation(
         // For now it only uses the first tag of the policy, since multi-tags are not yet supported
         const tagListKey = _.first(_.keys(policyTags));
         const uniquePolicyRecentlyUsedTags = recentlyUsedPolicyTags ? _.filter(recentlyUsedPolicyTags[tagListKey], (recentlyUsedPolicyTag) => recentlyUsedPolicyTag !== tag) : [];
-        optimisticPolicyRecentlyUsedTags[tagListKey] = [tag, ...uniquePolicyRecentlyUsedTags];
+        if (tag) {
+            uniquePolicyRecentlyUsedTags.unshift(tag);
+        }
+        optimisticPolicyRecentlyUsedTags[tagListKey] = [...uniquePolicyRecentlyUsedTags];
```

Comment:
Why are tags being updated? This doesn't match the [proposal](https://github.com/Expensify/App/issues/28988#issuecomment-1750405992) where it's `optimisticPolicyRecentlyUsedCategories` that are supposed to be updated in this file.